go.mod

```
@@ -9,7 +9,7 @@ require (
 
 require (
 	github.com/opencontainers/go-digest v1.0.0 // indirect
-	github.com/opencontainers/image-spec v1.1.0-rc2 // indirect
-	golang.org/x/sync v0.1.0 // indirect
-	golang.org/x/sys v0.6.0 // indirect
+	github.com/opencontainers/image-spec v1.1.0-rc3 // indirect
```

Could you use more SemVer one?
```suggestion
	github.com/opencontainers/image-spec v1.1.0-rc.3 // indirect
```

---

Oh! Fixed.